src/functions/utils/labels.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Utility functions for assessing the values of various post type labels.
+ */
+
+/**
+ * A simple function for knowing if the "Event"/"Events" label has been changed.
+ *
+ * @since TBD
+ *
+ * @return bool
+ */
+function tribe_is_event_label_customized() {
+
+	if (
+		'Event' !== tribe_get_event_label_singular() ||
```

Comment:
* This will probably work in most cases but I'll be a nerd and point out this would flop if filter functions for the corresponding hooks are not used throughout the whole request (ie, if someone listens for the calendar widget firing, and applies then unhooks one of those filters - to run once for a special effect, as it were - the result of this function wouldn't be reliable ... but I accept that's an edge case)
* We could probably simplify and condense to `return ( $cond_a || $cond_b /* || ... */ )` and drop the `if()` structure

Reply:
that's a great point. i agree the failure case is an edge-case but definitely something we should improve upon if we think of it